File: python/uri1013.py
import math
A, B, C = map(int, input().split(' '))

maiorab = (A + B + abs(A - B)) / 2

if (maiorab > C):
    print("{:.0f} eh maior". format(maiorab))
else:
    print("{} eh maior". format(C))

File: c++/uri1013.cpp
#include <iostream>
#include <cmath> 
using namespace std;
 
int main() {
 
    int A, B, C;
    cin >> A >> B >> C;
    int maiorAB = ((A + B + abs( A - B)) / 2);
    if (maiorAB > C) {
        cout << maiorAB << " eh o maior" << endl;
    } else {
        cout << C << " eh o maior" << endl;
    }
    return 0;
}